common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -2147,7 +2147,6 @@ schematic = (function() {
 	    this.bg_image.height = this.height;
 
 	    if (!this.diagram_only) {
-		this.canvas.tabIndex = 1; // so we get keystrokes
```

Comment:
@clrux notice the comment on this line.  The author added the tabIndex so that the canvas can receive keyboard events. I'm not familiar enough with this control to know *what* keyboard events they are expecting to receive and process, but the comment indicates to me that there are some.  If this were a fully accessible canvas widget, there would be focusable *fallback* content as a child of the canvas element.  Ideally, that content would be receiving the keyboard events.  However, without fallback content, canvas elements will not receive keyboard events.

My recommendation, until we fix this up properly, is to change this to a tabIndex = 0 so that it is in the natural tab order and NOT the 1st focusable element on the page (especially since there can be two or more of these widgets on a page.)